Tests/unit/core/test_time.cpp

```diff
@@ -21,177 +21,1637 @@
 
 #include <gtest/gtest.h>
 #include <core/core.h>
+#include <ctime>
 
 using namespace WPEFramework;
 using namespace WPEFramework::Core;
+static constexpr uint32_t MilliSecondsPerSecond = 1000;
+static constexpr uint32_t MicroSecondsPerMilliSecond = 1000;
+static constexpr uint32_t MicroSecondsPerSecond = MilliSecondsPerSecond * MicroSecondsPerMilliSecond;
 
-TEST(Core_Time, timedetails)
+
+int32_t GetMachineTimeDifference(string zone)
```

Comment:
I understand it is only testcode so just as a note const string& is more performant when passing string (or other complex parameters in general) as then no copying of the content needs to be done

Reply:
done